.github/workflows/wheels.yml

```diff
@@ -0,0 +1,276 @@
+          pip uninstall -y msprime
+          pip install -v msprime --only-binary msprime -f dist/wheelhouse --no-index
+          python -c "import msprime"
+          pip install -r requirements/tests-pypi.txt
+          # We can't install stdpopsim normally as it would try to compile msprime....
+          pip install stdpopsim pyslim kastore appdirs --no-deps
+          python -m nose -vs --processes=`nproc` --process-timeout=5000
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Run tests (3.7)
+        run: |
+          python -VV
+          # We install in this odd way to make sure we get both deps and a local msprime
+          pip install msprime --only-binary msprime -f dist/wheelhouse
+          pip uninstall -y msprime
+          pip install -v msprime --only-binary msprime -f dist/wheelhouse --no-index
+          python -c "import msprime"
+          pip install -r requirements/tests-pypi.txt
+          # We can't install stdpopsim normally as it would try to compile msprime....
+          pip install stdpopsim pyslim kastore appdirs --no-deps
+          python -m nose -vs --processes=`nproc` --process-timeout=5000
+
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+
+      - name: Run tests (3.6)
+        run: |
+          python -VV
+          # We install in this odd way to make sure we get both deps and a local msprime
+          pip install msprime --only-binary msprime -f dist/wheelhouse
+          pip uninstall -y msprime
+          pip install -v msprime --only-binary msprime -f dist/wheelhouse --no-index
+          python -c "import msprime"
+          pip install -r requirements/tests-pypi.txt
+          # We can't install stdpopsim normally as it would try to compile msprime....
+          pip install stdpopsim pyslim kastore appdirs --no-deps
+          python -m nose -vs --processes=`nproc` --process-timeout=5000
+      - name: Upload Wheels
+        uses: actions/upload-artifact@v2
+        with:
+          name: linux-wheels
+          path: dist/wheelhouse
+
+
+  PyPI_Upload:
+    runs-on: ubuntu-latest
+#    needs: ['windows', 'OSX', 'manylinux']
+    needs: ['manylinux']
+    steps:
+      - name: Download all
+        uses: actions/download-artifact@v2
+      - name: Move to dist
+        run: |
+          mkdir dist
+          cp */*.{whl,gz} dist/.
+      - name: Publish distribution to Test PyPI
+        if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags') && !contains(github.event.ref, 'C_')
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          password: ${{ secrets.test_pypi_password }}
+          repository_url: https://test.pypi.org/legacy/
+      - name: Publish distribution to PRODUCTION PyPI
+        if: github.event_name == 'release' && !startsWith(github.event.release.tag_name, 'C_')
```

Comment:
We can get rid of the ``startswith C_`` bit here - there'll never be C releases of msprime. There's  couple more instances of this elsewhere I think.